forte/data/ontology/top.py

```diff
@@ -1168,12 +1170,74 @@ def __init__(
         )
 
 
+class Payload(Entry):
+    def __init__(
+        self,
+        pack: PackType,
+        modality: str,
+        payload_idx: int,
+    ):
+        self.payload_idx = payload_idx
+        self.modality = modality
+        super().__init__(pack)
```

Comment:
It's because `super().__init__(pack)` will use those class variables defined before it. Similar thing happened to [Annnotation](https://github.com/asyml/forte/blob/ac34f102ebb3319c1e23f2f81e20a7e9e6c5e60d/forte/data/ontology/top.py#L81)

Reply:
hmm, so these are inevitable.

btw, we should make most variables private.

Reply:
I have made those variables private.